### PR TITLE
Remove "data" directory as submodules

### PR DESCRIPTION
**Pull request by cpaxton**
## Motivation and Context

This made cloning with `--recursive` very slow and wasnt necessary for people interested mostly in hardware and world representation.

## Changelog

- create `download_data.sh` script for cloning data repos
- change install_deps to use this

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

**Comment by Jdvakil** [comment]
LGTM

File: download_data.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	esac
+fi
+
+cd $HOME_ROBOT_ROOT
+
+# Create data, do not error if it already exists
+mkdir -p data/datasets
+
+cd $HOME_ROBOT_ROOT/data
+git clone https://huggingface.co/datasets/hssd/hssd-hab --recursive
```
Comment (yvsriram) [inline download_data.sh:51]:
```suggestion
git clone https://huggingface.co/datasets/hssd/hssd-hab --recursive --branch ovmm
```

File: download_data.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	esac
+fi
+
+cd $HOME_ROBOT_ROOT
+
+# Create data, do not error if it already exists
+mkdir -p data/datasets
+
+cd $HOME_ROBOT_ROOT/data
+git clone https://huggingface.co/datasets/hssd/hssd-hab --recursive --branch ovmm
```
Comment (yvsriram) [inline download_data.sh:51]:
```suggestion
git clone https://huggingface.co/datasets/hssd/hssd-hab --recursive --branch ovmm
cd ovmm
git checkout 4e0292b16b32c92267be1ac8a94628deccd2ef09

cd $HOME_ROBOT_ROOT/data
```

File: download_data.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+git clone https://huggingface.co/datasets/ai-habitat/OVMM_objects objects --recursive
```
Comment (yvsriram) [inline download_data.sh:52]:
```suggestion
git clone https://huggingface.co/datasets/ai-habitat/OVMM_objects objects --recursive

cd objects
git checkout 3893a735352b92d46505f35d759553f5fc82a39b
```

File: download_data.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+cd $HOME_ROBOT_ROOT/data/datasets
+git clone https://huggingface.co/datasets/ai-habitat/OVMM_episodes ovmm --recursive
```
Comment (yvsriram) [inline download_data.sh:55]:
```suggestion
git clone https://huggingface.co/datasets/ai-habitat/OVMM_episodes ovmm --recursive
cd ovmm
git checkout 9ad25fbd86a3fd352c7a0fc1f99132fbb5802378
```

**Review by yvsriram — APPROVED** [review]
We may want to checkout to the commits. Otherwise, looks good if it works.

**Comment by yvsriram** [comment]
I think .gitmodules should have been updated as well? I will manually update this.

Also documentation will need updates: projects/habitat_ovmm/README.md, src/home_robot_sim/install.sh, projects/habitat_ovmm/install.sh

**Review by yvsriram — APPROVED** [review]
The linting update looks good to me. contact_grasp_server and lseg kept causing  issues at times. Thanks for looking into this!

**Pull request merged**